flow/raster_cache.cc

```diff
@@ -405,9 +412,25 @@ void RasterCache::SweepOneCacheAfterFrame(RasterCacheKey::Map<Entry>& cache,
 
   for (auto it = cache.begin(); it != cache.end(); ++it) {
     Entry& entry = it->second;
-
     if (!entry.used_this_frame) {
-      dead.push_back(it);
+      if (entry.unused_count < entry.unused_threshold()) {
+        entry.unused_count++;
+        if (entry.image) {
+          RasterCacheKeyKind kind = it->first.kind();
```

Comment:
Maybe have a function `metrics_for(kind)` or `metrics_for(<type of it.first>)` to simplify all 3 cases where we have to choose a metrics to modify?

Reply:
done, added a new method `GetMetricsForKind`.